project/pyproject.toml.jinja

```diff
@@ -10,9 +10,9 @@ homepage = "https://[[ repository_provider ]]/[[ repository_namespace ]]/[[ repo
 keywords = []
 packages = [ { include = "[[ python_package_import_name ]]", from = "src" } ]
 classifiers = [
-    "Framework :: VOLTTRON",
-    "Framework :: VOLTTRON :: Agent",
-    "Framework :: VOLTTRON :: 0.1.0"
+    "Programming Language :: Python :: 3 :: Only",
+    "Intended Audience :: Science/Research",
```

Comment:
I am not sure about the intended audience. Volttron is used by commercial companies too. so not science/research alone. Can we specify multiple?  If not we can skip this one

Reply:
I'll add more audiences based on the list in https://pypi.org/classifiers/.